specification/workloads/Workloads.SAPMonitor.Management/models.tsp

```diff
@@ -671,3 +671,355 @@ model MsSqlServerProviderInstanceProperties extends ProviderSpecificProperties {
    */
   providerType: "MsSqlServer";
 }
+
+/**
+ * Gets or sets the Oracle provider properties.
+ */
+@added(Versions.v2024_02_01_preview)
+model OracleProviderInstanceProperties extends ProviderSpecificProperties {
+  /**
+   * Gets or sets the SQL server host name.
+   */
+  hostname?: string;
+
+  /**
+   * Gets or sets the database sql port.
+   */
+  dbPort?: string;
+
+  /**
+   * Gets or sets the database name.
+   */
+  dbName?: string;
+
+  /**
+   * Gets or sets the database user name.
+   */
+  dbUsername?: string;
+
+  /**
+   * Gets or sets the database password.
+   */
+  @secret
+  dbPassword?: string;
+
+  /**
+   * Gets or sets the key vault URI to secret with the database password.
+   */
+  dbPasswordUri?: string;
+
+  /**
+   * Gets or sets the SAP System Identifier
+   */
+  sapSid?: string;
+
+  /**
+   * Gets or sets certificate preference if secure communication is enabled.
+   */
+  sslPreference?: SslPreference;
+
+  /**
+   * Gets or sets the blob URI to SSL certificate for the SQL Database.
+   */
+  sslCertificateUri?: string;
+
+  /**
+   * The provider type. For example, the value can be SapHana.
+   */
+  providerType: "Oracle";
+}
+
+/**
+ * Describes the properties of an Alert for SAP monitor.
+ */
+@added(Versions.v2024_02_01_preview)
+model AlertProperties {
+  /**
+   * Defines the SAP monitor Alert errors.
+   */
+  @visibility("read")
+  errors?: Azure.ResourceManager.Foundations.ErrorDetail;
+
+  /**
+   * The ARM ID of the alert rule resource.
+   */
+  @visibility("read")
+  alertRuleResourceId?: string;
+
+  /**
+   * Defines the SAP monitor Alert Template Name.
+   */
+  templateName?: string;
+
+  /**
+   * Defines the provider type.
+   */
+  providerType?: string;
+
+  /**
+   * Defines the provider names.
+   */
+  providerNames?: string[];
+
+  /**
+   * Defines the alert rule properties.
+   */
+  alertRuleProperties?: AlertRuleProperties;
+
+  /**
+   * State of provisioning of the provider instance
+   */
+  @visibility("read")
+  provisioningState?: WorkloadMonitorProvisioningState;
+}
+
+/**
+ * Defines the Alert Rule Properties.
+ */
+@added(Versions.v2024_02_01_preview)
+model AlertRuleProperties {
+  /**
+   * Indicates whether the alert is in an enabled state.
+   */
+  status?: string;
+
+  /**
+   * Indicates the severity of the alert.
```

Comment:
>Indicates the severity of the alert. [](http://example.com/codeflow?start=5&length=36)

the comment here translate to "description" in swagger, which eventually go the the SDKs and client API documentation. 

So please make sure the descriptions in original swagger are included here without miss.